recipes/llvmdev38/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# (Mostly) blatant copy of https://github.com/numba/llvmlite/tree/master/conda-recipes/llvmdev
+
+{% set shortversion = "3.8" %}
+{% set version = "3.8.1" %}
+{% set sha256 = "6e82ce4adb54ff3afc18053d6981b6aed1406751b8742582ed50f04b5ab475f9" %}
+
+package:
+  name: llvmdev38
+  version: {{ version }}
+
+source:
+  fn: llvm-{{ version }}.src.tar.xz
+  url: http://llvm.org/releases/{{ version }}/llvm-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    # https://llvm.org/bugs/show_bug.cgi?id=15356
+    - llvm-elf-relocation.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25788
+    - llvm-parse-fp-constants.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25565
+    - llvm-build-race-condition.patch
+    # http://lists.llvm.org/pipermail/llvm-dev/2016-January/094520.html
+    - llvm-lto-static.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
As to other weird CMake parsing issues, see this case of `-D` being stripped.

xref: https://github.com/conda-forge/cmake-feedstock/pull/16#issuecomment-249730942